sql/5_0_2-to-5_0_3_upgrade.sql

```diff
@@ -419,4 +419,15 @@ CREATE TABLE `pro_assessments` (
 
 #IfNotRow2D list_options list_id LBF_Validations option_id future_date
 INSERT INTO `list_options` (`list_id`,`option_id`,`title`,`notes`, `seq`) VALUES ('LBF_Validations','future_date','Future Date','{\"futureDate\":{\"message\":\"must be future date\"}}','32');
+
+#IfNotRow2D list_options list_id lists option_id Sort_Direction
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `activity`) VALUES ('lists', 'Sort_Direction', 'Sort Direction', 306, 1, 1);
+#EndIf
+
+#IfNotRow2D list_options list_id Sort_Direction option_id 0
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `activity`) VALUES ('Sort_Direction', '0', 'asc', 1, 1, 1);
+#EndIf
+
+#IfNotRow2D list_options list_id Sort_Direction option_id 1
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `activity`) VALUES ('Sort_Direction', '1', 'desc', 2, 0, 1);
```

Comment:
for the seq on these, start at 10 and increment by 10 (this gives breathing room if wish to add other items in the future). Plus the asc/desc titles seem on the technical say. Would there be a less technical way to state these options (for folks that may not know was asc/desc mean (if not, then keep it way as is)

Reply:
Fixed